tests/Functional/App/routing.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<routes xmlns="http://symfony.com/schema/routing"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/routing http://symfony.com/schema/routing/routing-1.0.xsd">
+
+    <import resource="." type="content_api"/>
```

Comment:
This is annoying that you have to add something to your app's routing file. Can't find a way around it for the moment.

Reply:
It's probably useful to be explicit if someone has other routing configuration for separate controllers and can clearly separate between who handles what?

Reply:
Does allow you to put them being a prefix, if you're mixing a Libero API with other stuff.